sdk/tables/azure-tables/src/TableServiceClient.ts

```diff
@@ -133,13 +140,20 @@ export class TableServiceClient {
    * @param rowKey The row key of the entity.
    * @param options The options parameters.
    */
-  getEntity(
+  public async getEntity(
     tableName: string,
     partitionKey: string,
     rowKey: string,
     options?: GetEntityOptions
   ): Promise<GetEntityResponse> {
-    return this.table.queryEntitiesWithPartitionAndRowKey(tableName, partitionKey, rowKey, options);
+    const response = await this.table.queryEntitiesWithPartitionAndRowKey(
+      tableName,
+      partitionKey,
+      rowKey,
+      options
+    );
+    response.value = deserialize(response._response.parsedBody) as object[];
```

Comment:
Just wondering, how do we know this `as object[]` thing here is safe?  It doesn't seem like `deserialize` would return an array like thing?

Reply:
You're right. The problem initially was that the swagger had defined an array of objects for the response of getEntity but the service was returning just the object itself and not as part of an array, so I just cast it to get rid of TS error for now.

It seems like they just fixed the swaggers, so I'll try to regenerate and get rid of the cast.